azure-pipelines-ci.yml

```diff
@@ -92,6 +105,22 @@ stages:
         summaryFileLocation: 'build/reports/jacoco/jacocoRootReport/*.xml'
         reportDirectory: 'build/reports/jacoco/jacocoRootReport/html'
       condition: eq(variables.fullBuild, 'false')
+    - task: Gradle@2
+      displayName: Package Android client APK
+      inputs:
+        workingDirectory: 'android_client'
+        gradleWrapperFile: 'android_client/gradlew'
+        tasks: 'assembleRelease'
+        publishJUnitResults: false
+        javaHomeOption: 'JDKVersion'
+        jdkVersionOption: '1.11'
+        options: '-PclientVersionCode=$(hydraVersionCode) -PclientVersionNumber=$(hydraVersion)'
+    - task: PowerShell@2
+      displayName: Copy APK to folder for deployment
+      inputs:
+        targetType: 'inline'
+        script: |
+          cp android_client/app/build/outputs/apk/release/app-release.apk common/src/main/java/record_release.apk -force
```

Comment:
Why copy apk to this location? Shouldn't it be common/src/main/resources/record_release.apk?

Besides, can you also copy the androidTest apk to the same folder as both the apks would be leveraged in the Center/Uber images as the default app to be tested.